### ISSUE TITLE
Effects of BatchNorm

### ISSUE THREAD

**Issue by chengdazhi** [ChiWeiHsiao/DeepVO_PyTorch#29]
Hi Chiwei, thanks for sharing your code, which is very helpful.

I was looking into it and saw that you are using BatchNorm. I don't see the DeepVO mentioning it. Did you do an experiment on this?

Also, why did you use more training data than the settings in the paper? And did you evaluate the RMSE of your model?

Thank you very much!
